app/src/main/java/com/practicum/playlistmaker/PlayerActivity.kt

```diff
@@ -1,38 +1,54 @@
 package com.practicum.playlistmaker
 
+import android.icu.text.SimpleDateFormat
+import android.media.MediaPlayer
 import android.os.Bundle
-import android.widget.ImageView
-import android.widget.TextView
+import android.os.Handler
+import android.os.Looper
 import androidx.appcompat.app.AppCompatActivity
-import androidx.appcompat.widget.Toolbar
 import com.bumptech.glide.Glide
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners
-import java.text.SimpleDateFormat
+import com.practicum.playlistmaker.databinding.ActivityPlayerBinding
 import java.util.Calendar
 import java.util.Date
 import java.util.Locale
 
 class PlayerActivity : AppCompatActivity() {
 
+    companion object {
+        private const val STATE_DEFAULT = 0
+        private const val STATE_PREPARED = 1
+        private const val STATE_PLAYING = 2
+        private const val STATE_PAUSED = 3
+        private const val DELAY = 300L
+        private const val TIME_OFFSET = 500L
+    }
+
+    private var mainThreadHandler: Handler? = null
     lateinit var track: Track
+    private lateinit var binding: ActivityPlayerBinding
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Медиаплеер пересоздавать не нужно, это большой объект и имеет своё состояние, поэтому лучше переиспользовать существующий инстанс и не пересоздавать его